tensorboard/webapp/core/effects/core_effects.ts

```diff
@@ -84,6 +87,33 @@ export class CoreEffects {
     {dispatch: false}
   );
 
+  /**
+   * HACK: COMPOSITE ACTION -- rationale: most plugins want to be able to tell
+   * when it becomes active in order to, for example, fetch necessary data. By
+   * firing changePlugin on activePlugin value set, we can prevent (1) other
```

Comment:
"By firing changePlugin on activePlugin value set"

Do you mean "By firing changePlugin when activePlugin value is first set?"  Or is this supposed to be a reference to the overall behavior of changePlugin: "By always firing changePlugin when activePlugin value is set".?